numpy/ma/extras.py

```diff
@@ -1137,15 +1137,22 @@ def setxor1d(ar1, ar2, assume_unique=False):
     flag2 = (flag[1:] == flag[:-1])
     return aux[flag2]
 
+
 def in1d(ar1, ar2, assume_unique=False, invert=False):
     """
     Test whether each element of an array is also present in a second
     array.
 
     The output is always a masked array. See `numpy.in1d` for more details.
 
+    This function has been deprecated: use `isin` instead.
+
+    Deprecated since version 1.13.0.
```

Comment:
Should use the `.. deprecated:: 1.13.0` syntax. Look around at other files to see how it's done